ingest.py

```diff
@@ -82,17 +88,38 @@ def ingest_docs():
     # with open('docs_transformed.pkl', 'rb') as f:
     #     docs_transformed = pickle.load(f)
     
-    client = weaviate.Client(url=WEAVIATE_URL, auth_client_secret=weaviate.AuthApiKey(api_key=WEAVIATE_API_KEY))
-    client.schema.delete_class("LangChain_newest_idx") # delete the class if it already exists
+    client = weaviate.Client(
+        url=WEAVIATE_URL,
+        auth_client_secret=weaviate.AuthApiKey(api_key=WEAVIATE_API_KEY)
+    )
+    embedding = OpenAIEmbeddings(chunk_size=200)  # rate limit
```

Comment:
why does this say rate limit?

Reply:
when upserting into weaviate, we ran into openai rate limits with default chunk sizes, so declaring it to be 200 helped deal with that